var/spack/repos/builtin/packages/phist/package.py

```diff
@@ -118,6 +120,8 @@ def cmake_args(self):
                 % ('ON' if '+trilinos' in spec else 'OFF'),
                 '-DPHIST_USE_PRECON_TPLS:BOOL=%s'
                 % ('ON' if '+trilinos' in spec else 'OFF'),
+                '-DXSDK_BUILD_Fortran:BOOL=%s'
```

Comment:
Will a `@:1.6.99` version of `phist` just ignore this argument? Or do you need to guard it with an if statement?